data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,370 @@
+			choice
+				`	(Sure.)`
+				`	(Maybe another time.)`
+					defer
+			`	You proceed to hire a motorized antigrav vehicle to go there, and by the time you arrive it starts being dark. You set foot after double checking you are indeed at the right coordinates, for except a cave, vegetation, sand, and, of course, the beach, you cannot see much of interest in the vicinity. You can't help but put your hand close to your gun, feeling safer with its presence.`
+			`	After waiting for what seems like an eternity you start to hear whisper, and two hooded figures approach you. You start to doubt whether or not this was a good idea but it is probably too late to back down now. They come to address you, their face hidden by the now almost total obscurity. "Hello, <last>," they say at the same time, almost in a comical way, removing some of the tension you felt until now. Then they look at each other, nod, and one of them addresses you in a more serious mood. "We are sorry for all of these complications, but the truth is we cannot afford to take risks. It has been some time since we stopped trying to understand, and started taking every precaution we could to be safe. You see, we simply do not know you, and neither do you us."`
+			choice
+				`	"Tell me more."`
+					goto tellme
+				`	"How do you know me?"`
+					goto know
+				`	(Run away).`
+			`	They seem surprised at your sudden change of mind, but make no effort to stop you. You proceed to take the antigrav back to the shipyard without any incidents.`
+				decline
+			label know
+			`	They grin, "Us knowing you is a necessity for you to land on our homeworld."`
+			label tellme
+			`	"We have a very interesting proposition for you: should you agree to bring supplies to our brethren, the Unfettered Hai, we will offer you a ship, for a modest price of 6.5 million credits. This is a very useful ship, and you should be able to repay this quite easily by running more of the same kind of jobs. It is indeed appropriate for bringing this important food cargo safely to destination, and even defend yourself, if need be. You see, these supplies are needed on short notice, on <destination>, and we lack the time and personnel to bring them."`
+			`	They take you into the cave, finally showing the ship they were referring to. "This is a mighty relic of the past, from the times our Empire used to control the stars; a Ladybug!`
+			scene "thumbnail/hai ladybug"
+			`	"Of course, it has some... drawbacks, compared to its golden time, and don't expect the best price in a shipyard either. Nevertheless it is a fine vessel, capable of safely taking a respectable amount of cargo across large distances! So, what do you say, <last>? This is the opportunity of a lifetime, as they simply don't make ships like these anymore!"`
+			choice
+				`	"I will take this deal."`
+					goto take
+				`	"I am not interest in this deal."`
```

Comment:
```suggestion
				`	"I am not interested in this deal."`
```

Reply:
Could also be `"I have no interest in this deal."`

Reply:
I like that one more
```suggestion
				`	"I have no interest in this deal."`
```